### PR TITLE
[ELY-2455]: Add a "FAQ for New Contributors" section to the CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by harshittiwariii**
[ELY-2455]: Add a "FAQ for New Contributors" section to the CONTRIBUTING.md file

Issue: https://issues.redhat.com/browse/ELY-2455

-Added FAQ's section for New contributors.
-Updated the CONTRUBITING.md file.
  -Pull Request section added for more clarity.
  -Testing and Documentations guidelines added for new contributors to understand the importance of documentation and 
    testing their fixes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,8 +89,24 @@ Note: Some tests will fail if `localhost` is not listed first in `/etc/hosts` fi
...
+
+Ensure that your changes are thoroughly tested before submitting a pull request. Follow these testing guidelines:
+
+- Run the existing unit tests using Maven: `mvn clean test`
+- Include new unit tests for your code changes.
+
+If applicable, provide instructions on how to run integration tests or any additional testing procedures.
+
+### Documentation
+Contributors are encouraged to keep documentation up-to-date along with code changes. If your changes impact user-facing features, update the relevant documentation files in directory.
```
Comment (skyllarr) [inline CONTRIBUTING.md:105]: @harshittiwariii Just a minor, community documentation can be found in the wildfly repository https://github.com/wildfly/wildfly/tree/main/docs

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -97,5 +117,35 @@ When submitting a PR, please keep the following guidelines in mind:
 
 For an example of a properly formatted PR, take a look at https://github.com/wildfly-security/wildfly-elytron/pull/1532
 
+## FAQ for New Contributors
+
+### Q: How do I find good first issues to work on?
```
Comment (skyllarr) [inline CONTRIBUTING.md:122]: @harshittiwariii Just a minor, it is sufficient to include a link to https://fjuma.github.io/wildfly-elytron/blog/frequently-asked-questions-new-contributors/ page in this section

**Review by skyllarr — COMMENTED** [review]
Thank you @harshittiwariii ! I added some comments. Please make sure to include an issue number at the beginning of your commit message and make sure there is just 1 commit by squashing the commits. Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (harshittiwariii) [reply CONTRIBUTING.md:105]: [ELY-2455]
@Skyllarr  Updated and requested for review.

File: CONTRIBUTING.md (hunk 2 above)
Comment (harshittiwariii) [reply CONTRIBUTING.md:122]: [ELY-2455]
@Skyllarr  Updated and requested for review.

**Comment by harshittiwariii** [comment]
close for version

**Pull request closed without merging**